pom.xml

```diff
@@ -221,6 +222,13 @@
       <artifactId>java-hamcrest</artifactId>
       <scope>test</scope>
     </dependency>
+
+    <!-- Needed to fix @Generated annotation java 8 - 11 issue, Javadoc generation with JDK 9 -->
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>${javax-annotation-api.version}</version>
```

Comment:
Can we have it as `provided` — not as a compile and runtime dependency?

Reply:
I have some problems when using it as `provided` - ### `archetype` module compiles with fails under Java 11 for some reason.

Reply:
```
 Caused by: org.apache.maven.plugin.compiler.CompilationFailureException: Compilation failure
[INFO] Bad service configuration file, or exception thrown while constructing Processor object: javax.annotation.processing.Processor: Provider com.ryanharter.auto.value.gson.AutoValueGsonAdapterFactoryProcessor could not be instantiated
```